src/redux/configureStore.js

```diff
@@ -0,0 +1,11 @@
+import { combineReducers, createStore } from 'redux';
```

Comment:
- [ OPTIONAL] It would be nice to implement the project requirement by using `configureStore` provided by `@reduxjs/toolkit` instead of using `createStore` to create the global store. 👍 

### Project Requirement

![image](https://user-images.githubusercontent.com/77106962/210833392-76b4cc39-8cc8-4736-82a2-9ad7172732dc.png)

#### Hint: You can learn how to use configureStore by reading this [article](https://redux.js.org/introduction/why-rtk-is-redux-today#:~:text=configureStore%20sets%20up%20a%20well,it%20takes%20named%20options%20parameters.) 💻